stats_test.go

```diff
@@ -208,3 +235,155 @@ func TestNilAggregation(t *testing.T) {
 		t.Errorf("Expected: %v, Got: %v", fmt.Errorf("aggregation *view.Aggregation is not supported"), actual)
 	}
 }
+
+func Test_calculatePercentile(t *testing.T) {
```

Comment:
Same here and the test below. Can we not use `_` underscore?

Reply:
Fixed.